deb/build/debian/jenkins.service

```diff
@@ -0,0 +1,24 @@
+[Unit]
+Description=Jenkins Continuous Integration Server
+After=network.target
+
+[Service]
+Type=simple
+User=jenkins
+Group=jenkins
+SyslogIdentifier=jenkins
+UMask='0002'
+LimitNOFILE='8192'
+
+Environment=JENKINS_HOME=/var/lib/jenkins
+ExecStart=/usr/bin/java \
+    -jar /usr/share/jenkins/jenkins.war \
+    --webroot=/var/cache/jenkins/war \
+    --pluginroot=/var/cache/jenkins/plugins \
+    --httpPort='8080' \
+
+# We send SIGTERM(15), java exit code is 143 = 128 + 15
+SuccessExitStatus=143
```

Comment:
Jenkins does not handle Unix signals, that ends up killing the application and Java exit code is `128 + signal number`.   Killing it with a `SIGINT` would just cause java to exit with `130`.

The shell based script `deb/build/debian/jenkins.init` uses `daemon --stop` which emits `SIGTERM`.